spago.dhall

```diff
@@ -1,7 +1,14 @@
 { name = "elmish-html"
 , dependencies =
-    [ "elmish"
+    [ "effect"
+    , "elmish"
     , "foreign-object"
+    , "foreign"
+    , "prelude"
+    , "record"
+    , "typelevel-prelude"
+    , "unsafe-coerce"
+    , "web-html"
```

Comment:
This came from upgrading Spago: the new version errors out when there are referenced, but undeclared depdendencies.